client/src/modules/user/AuthBox/components/SignInForm.jsx

```diff
@@ -61,11 +83,18 @@ class SignInForm extends PureComponent {
     );
   };
 
+  onPolicyChange = () =>
+    this.setState(({ isPolicyAccepted }) => ({
+      isPolicyAccepted: !isPolicyAccepted
+    }));
```

Comment:
I would go for 

`acceptPolicy = () => this.setState({isPolicyAccepted: true});`

`dismissPolicy = () => this.setState({isPolicyAccepted: false});`